tsconfig.json

```diff
@@ -7,8 +7,8 @@
         ],
         "module": "Node16",
         "outDir": "./out",
-        "sourceMap": true,
-        "target": "esnext",
+        "sourceMap": false,
```

Comment:
Why did we disable sourceMaps again when merging main? Don't we need those for debugging?

Reply:
When merging with the main branch, I noticed it was set to false, so I assumed that was the intended setting. However, on my local machine, I had set it to true

Reply:
I think you need true to properly debug the extension. For releases and packaging the vsix it should be false. How is this usually handled? Do we want a default for releases or for development?